src/python/grpcio_tests/tests/protoc_plugin/beta_python_plugin_test.py

```diff
@@ -57,12 +57,18 @@
         'service',
```

Comment:
Sorry. Not the right file. As the filename indicates, these tests are for the `beta` protoc (for pre gRPC 1.0 generated code). You'll want to put this test in `_split_definitions_test.py` or a new test file.

Reply:
Ahha. I didn't want to edit a file starting with an underscore, so I picked the only non-underscore file to edit. I've moved it to `_split_definitions_test`.